### PR TITLE
webui: improve selenium test

### PR DESCRIPTION
**Pull request by joergsteffens**
Low resolution displays the top level menu as a sandwich menu.
This change can now handle both display forms of the top level menu.

### CODE REVIEW

**Comment by fbergkemper** [comment]
This branch has conflicts that must be resolved

File: webui/tests/selenium/webui-selenium-test.py [hunk 1]
```diff
@@ -196,6 +196,13 @@ def setUp(self):
                 self.driver = webdriver.Firefox(capabilities=d, firefox_profile=fp)
             else:
                 raise RuntimeError('Browser {} not found.'.format(str(self.browser)))
+            #
+            # set explicit window size
+            #
+            # Used by Univention AppCenter test.
+            self.driver.set_window_size(1200, 800)
```
Comment (fbergkemper) [inline webui/tests/selenium/webui-selenium-test.py]: Making the resolution the test runs with configurable via environment variable in the testrunner would be great.

Comment (joergsteffens) [reply webui/tests/selenium/webui-selenium-test.py]: Implemented. Use BAREOS_WEBUI_BROWSER_RESOLUTION variable.

File: webui/tests/selenium/webui-selenium-test.py [hunk 2]
```diff
@@ -412,10 +416,31 @@ def login(self):
             raise WrongCredentialsException(self.username, self.password)
 
     def logout(self):
-        self.wait_and_click(By.CSS_SELECTOR, 'span.glyphicon.glyphicon-user')
+        self.wait_and_click(By.CSS_SELECTOR, 'span.glyphicon.glyphicon-user', [(By.CSS_SELECTOR, 'div.navbar-header > button')])
         self.wait_and_click(By.LINK_TEXT, 'Logout')
         sleep(self.sleeptime)
 
+    def select_tab(self, tab, additional_modals=None):
```
[the remark this answers is not in the document]
Comment (joergsteffens) [reply webui/tests/selenium/webui-selenium-test.py]: Renamed to select_navbar_element.

**Comment by joergsteffens** [comment]
My last commit also fixes the problem, that selenium failures have not resulted in test failures.

**Pull request merged**